scheduler/routes.py

```diff
@@ -53,7 +57,7 @@ async def websocket_handler(websocket: WebSocket) -> None:
             task = asyncio.to_thread(worker, data)
             await manager.send({"type": "update", "payload": {"message": "Processing plans..."}}, websocket)
             result = await task
-            await manager.send({"type": "plans", "payload": result}, websocket)
+            await manager.send({"type": "plans", "payload": json.dumps(result)}, websocket)
```

Comment:
Will json.dumps serialize `result` as json?

Reply:
Yes, that's the idea.